modules/mri_violations/php/mri_violations.class.inc

```diff
@@ -314,19 +362,24 @@ class Mri_Violations extends \NDB_Menu_Filter_Form
         // Get sites
         if ($user->hasPermission('access_all_profiles')) {
```

Comment:
same here, need to change the permission to be the violation_view all sites one

Reply:
Correct